### PR TITLE
Update to Ember 3.4

### PR DESCRIPTION
**Pull request by GreatWizard**
Just to stay on page 👍

### CODE REVIEW

**Comment by GreatWizard** [comment]
Tests are green and I need 1 review to merge it :)

File: .travis.yml [hunk 1]
```diff
@@ -12,37 +12,50 @@ addons:
...
-    - EMBER_TRY_SCENARIO=ember-canary
-    - EMBER_TRY_SCENARIO=ember-default
-
-matrix:
-  fast_finish: true
-  allow_failures:
+    - stage: "Additional Tests"
+      env: EMBER_TRY_SCENARIO=ember-lts-2.16
+    - env: EMBER_TRY_SCENARIO=ember-lts-2.18
+    - env: EMBER_TRY_SCENARIO=ember-release
```
Comment (mathieupoteriepeopledoc) [inline .travis.yml]: You don't use matrix anymore ?

File: README.md [hunk 2]
```diff
@@ -45,17 +46,17 @@ You can configure a route for feature-controls in your app. This will add a rout
 
 ```js
 // app/router.js
-import EmberRouter from '@ember/routing/router';
-import config from './config/environment';
-import featureControlsRouteSetup from 'ember-feature-controls/route-setup';
+import EmberRouter from "@ember/routing/router";
```
Comment (mathieupoteriepeopledoc) [inline README.md]: is that normal that you get double quotes everywhere ?

File: .travis.yml (hunk 1 above)
Comment (GreatWizard) [reply .travis.yml]: This project used the standard travis CI configuration.
It's already a tests matrix configuration :)

File: addon/templates/components/feature-controls.hbs [hunk 3]
```diff
@@ -25,9 +30,9 @@
 </table>
 
 {{#if showRefresh}}
-  <button {{action 'refresh'}} data-test-button-refresh>Refresh</button>
+  <button {{action "refresh"}} data-test-button-refresh>Refresh</button>
```
Comment (mathieupoteriepeopledoc) [inline addon/templates/components/feature-controls.hbs]: here I supposed this is normal to get double quotes (eslint with .hbs files)

File: README.md (hunk 2 above)
Comment (GreatWizard) [reply README.md]: Yes it's the default configuration for eslint for an ember-project. We are not using peopledoc config here.

File: addon/templates/components/feature-controls.hbs (hunk 3 above)
Comment (GreatWizard) [reply addon/templates/components/feature-controls.hbs]: Yes it's the new template linter that force that by default

**Pull request merged**